src/west/app/project.py

```diff
@@ -792,9 +794,10 @@ def do_run(self, args, ignored):
         for project in self._cloned_projects(args, only_active=not args.all):
             # Use paths that are relative to the base directory to make it
             # easier to see where the changes are
+            merge_base = ['--merge-base', project.revision] if args.manifest else []
```

Comment:
`project.revision` is not OK: it can be a _remote_ branch name and then `git diff` will fail. Or something else and who knows what might happen.

I just submitted a replacement of `project.revision` by `manifest-rev` in #748

Longer story in #747